Description: do not use obsolete compatibility file
Author: Martin <debacle@debian.org>
Origin: vendor
Last-Update: 2024-06-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/test_path.py
+++ b/tests/test_path.py
@@ -8,7 +8,7 @@
 import unittest
 from .compat.overlay import zipfile
 
-from .compat.py39.os_helper import temp_dir, FakePath
+from test.support.os_helper import temp_dir, FakePath
 
 import jaraco.itertools
 from jaraco.functools import compose
--- a/tests/compat/py39.py
+++ /dev/null
@@ -1,11 +0,0 @@
-import sys
-
-from jaraco.test.cpython import from_test_support, try_import
-
-
-os_helper = try_import('os_helper') or from_test_support(
-    'FakePath',
-    'temp_dir',
-)
-
-sys.modules[__name__ + '.os_helper'] = os_helper
--- a/tests/compat/py38.py
+++ /dev/null
@@ -1,10 +0,0 @@
-import sys
-
-
-if sys.version_info >= (3, 9):
-    from importlib.abc import Traversable
-else:  # pragma: no cover
-    from importlib_resources.abc import Traversable
-
-
-__all__ = ['Traversable']
